classes/abstracts/ActionScheduler_Lock.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * Class 
```

Comment:
```suggestion
 * Class
```

Remove trailing space. Should we look at getting a code sniffer set up for this repo?

Reply:
> Should we look at getting a code sniffer set up for this repo?

That would be great. The question on that is which set of standards should it follow? It definitely doesn't follow WP standards yet.